games-arcade/tuxpuck/tuxpuck-0.8.2.recipe

```diff
@@ -0,0 +1,60 @@
+SUMMARY="Airhockey with tux (Shufflepuck Café clone)"
+DESCRIPTION="Open source Shufflepuck Café clone, with tux and arcana"
+HOMEPAGE="https://wiki.debian.org/Games/TuxPuck"
+COPYRIGHT="2000-2002 Jacob Kroon"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="http://ftp.de.debian.org/debian/pool/main/t/tuxpuck/tuxpuck_$portVersion.orig.tar.gz"
+CHECKSUM_SHA256="62d9604ed69c27b9ca2be1312bc705b36de8ed509c539c6d81193e7846272f18"
+SOURCE_DIR="tuxpuck-$portVersion"
+PATCHES="buffer-overflow.patch
+	clang-FTBFS.patch
+	FTBFS-with-fread.patch
+	libpng-transition.patch 
+	Makefile.patch
+	cross.patch
+	haiku.patch"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	tuxpuck$secondaryArchSuffix = $portVersion
+	app:Tuxpuck = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libSDL_1.2$secondaryArchSuffix
+	lib:libfreetype$secondaryArchSuffix
+	lib:libjpeg$secondaryArchSuffix
+	lib:libpng16$secondaryArchSuffix
+	lib:libvorbis$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libSDL_1.2$secondaryArchSuffix
+	devel:libfreetype$secondaryArchSuffix
+	devel:libjpeg$secondaryArchSuffix
+	devel:libpng$secondaryArchSuffix
+	devel:libvorbis$secondaryArchSuffix
+	"
+
```

Comment:
Sorry, missed this one :)
No empty line between BUILD_REQUIRES and BUILD_PREREQUIRES
```diff
-
```

Reply:
Information about this can be found here: https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines